### PR TITLE
Upgrade to linter v1.31.0

### PR DESCRIPTION
**Pull request by mukundansundar**
# Description

Upgrade to linter v1.30.0

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #443 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Review by amanbha — DISMISSED** [review]

**Comment by artursouza** [comment]
Please, resolve conflicts.

**Review by youngbupark — DISMISSED** [review]

**Comment by mukundansundar** [comment]
UTs will be added by a separate issue #423

File: .github/workflows/dapr_cli.yaml
Comment (youngbupark) [inline .github/workflows/dapr_cli.yaml:25]: will you add linter action step as well?

**Review by youngbupark — APPROVED** [review]
👍

File: .github/workflows/dapr_cli.yaml
Comment (mukundansundar) [reply .github/workflows/dapr_cli.yaml:25]: will do ....

**Review by youngbupark — APPROVED** [review]

**Comment by tcnghia** [comment]
Thanks for making this change & clean up the code :).

I've noticed that the CodeCov/patch marked the PR as broken (red ❌). I sent https://github.com/dapr/cli/pull/482 to fix that. If you want to merge with a green ✅ you can wait until that PR is merged and pull from master

**Pull request merged**
